scripts/gulp/icons.js

```diff
@@ -27,6 +36,23 @@ function pfIconFont() {
         timestamp: Math.round(Date.now() / 1000)
       })
     )
+    // gulp-iconfont emits a 'glyphs' event
+    // we use to update existingIconCodes file
+    .on('glyphs', function(glyphs, options) {
+      const newIconsObj = glyphs.reduce((iconsObj, glyph) => {
+        const { name, unicode } = glyph;
+        const codePoint = unicode[0].codePointAt(0);
+        iconsObj[name] = codePoint.toString(16).toUpperCase();
+        iconsObj.codePoints.push(codePoint);
+        return iconsObj;
+      }, { codePoints: [] });
+
+      fs.writeFile(
+        path.join(process.cwd(), '/src/icons/existingIconCodes.json'),
```

Comment:
Use `path.join(__dirname, '../../src/icons/existingIconCodes.json'` so the cwd can be anywhere.